reroll-initiative/reroll-initiative.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * @name Reroll-Initiative
+ * @version 0.1
+ * @author Evan Clarke <errational>
+ * @description Rerolls initiative on combat round change
+ */
+
+/**
+ * Hook on game ready and then execute the module function
+ */
+Hooks.on("ready", () => {
+    rerollInitiative();
+});
+
+/**
+ * Main module function
+ */
+function rerollInitiative() {
+    console.log("reroll-initiative starting up...")
+    
+    const MODULE_NAME = "reroll-initiative";
```

Comment:
You define this constants multiple times for the same string. Why not have a class that has static methods to return this string names instead or a class with all these constants you can access?

Reply:
Can you please clarify this comment?
Regarding classes: this design pattern is meant to be classless and simple.

Reply:
After the latest change, this comment is no longer relevant.